execution/gethexec/sequencer.go

```diff
@@ -972,7 +928,15 @@ func (s *Sequencer) createBlock(ctx context.Context) (returnValue bool) {
 	}
 
 	start := time.Now()
-	block, err := s.execEngine.SequenceTransactions(header, txes, hooks)
+	var (
+		block *types.Block
+		err   error
+	)
+	if s.enableProfiling {
```

Comment:
Can we use `config.EnableProfiling` here instead? Then we could make this option hot-reloadable.

Reply:
Good idea! Done.